### PR TITLE
ci: builds with bundt :tada:

### PR DESCRIPTION
**Pull request by maraisr**

### CODE REVIEW

**Comment by theKashey** [comment]
🙄  ✅

File: __tests__/index.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { fireEvent, act, render, screen } from '@testing-library/react';
 import React, { ButtonHTMLAttributes, MutableRefObject } from 'react';
 
-import { HoverOptions, useHoverState } from '../src';
-import { HoveredStateTuple } from '../src/use-hover-state';
+import type { HoverOptions, HoveredStateTuple } from 'use-hover-state';
```
Comment (theKashey) [inline __tests__/index.tsx:4]: fancy self imports

File: tsconfig.json [hunk 2]
```diff
@@ -19,6 +19,9 @@
     "lib": ["dom", "es5", "scripthost", "es2015.collection", "es2015.symbol", "es2015.iterable", "es2015.promise"],
     "types": ["node", "jest"],
     "typeRoots": ["./node_modules/@types"],
-    "jsx": "react"
+    "jsx": "react",
+    "paths": {
+      "use-hover-state": ["./src/index.d.ts"]
```
Comment (theKashey) [inline tsconfig.json:24]: 🤯

File: package.json [hunk 3]
```diff
@@ -2,32 +2,36 @@
...
     "node": ">=10"
   },
   "scripts": {
-    "dev": "lib-builder dev",
     "test": "jest",
     "test:ci": "jest --runInBand --coverage",
-    "build": "lib-builder build && yarn size:report",
+    "build": "bundt --minify",
     "release": "yarn build && yarn test",
-    "size": "npx size-limit",
```
Comment (theKashey) [inline package.json:26]: this little guy is very important.

**Comment by theKashey** [comment]
Let's talk about underlaying bundler change separately, as well as the way one maintains types.

@thearnica - let's put this one on hold for a while

File: package.json (hunk 3 above)
Comment (maraisr) [reply package.json:26]: I put it back 🚀

**Pull request closed without merging**